modules/microshift-exposed-audit-ports-hostnetwork.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-exposed-audit-ports-hostnetwork_{context}"]
+
+== hostnetwork
+
+When a pod is configured with the `hostNetwork:true` setting, the pod is running in the host network namespace. This configuration can independently open host ports. {microshift-short} component logs cannot be used to track this case, the ports are subject to firewalld rules. If the port opens in firewalld, you can view the port opening in the firewalld debug log.
```

Comment:
How do I know if hostNetwork:true is set in my pod ?  

I am trying to use the following, but it says null
`oc get pod router-default-69f5966d4-5kq65 -n openshift-ingress -o json | jq -r '.spec.hostNetwork'`

Reply:
There are some build issues going on in the repo, but @pliurh do you know this one?

Reply:
That's expected. The command is correct. But the router pod in Microshift is not `hostNetwork:true`. The ovnkube-node pod is.
```
# oc get pod -n openshift-ovn-kubernetes   ovnkube-node-7mmlp  -o json | jq -r '.spec.hostNetwork'
true
```